static/sass/_snapcraft_p-video-wrapper.scss

```diff
@@ -0,0 +1,7 @@
+@mixin snapcraft-p-video-wrapper {
+  .p-video-wrapper {
```

Comment:
Can you use the Vanilla one? https://vanillaframework.io/docs/utilities/embedded-media